### PR TITLE
Make updates to references

### PR DESCRIPTION
**Pull request by souravsingh**
Fix for #150 

There are fixes needed for the AstroPy reference.

### CODE REVIEW

**Comment by souravsingh** [comment]
@mdhaber I have sent an initial PR for #150

**Comment by mdhaber** [comment]
@souravsingh thanks but it's not building. What is left to be done? Do you know when you'll finish?

**Comment by souravsingh** [comment]
I believe there are no changes needed( unless there are some). I will need to recheck the build failure, which could be happening due to a mistake somewhere in the bib file.

**Comment by ilayn** [comment]
The error says 

```bash
Database file #1: references.bib
I was expecting a `,' or a `}'---line 176 of file references.bib
 :   
 :   title={CSE 2009: Python for Scientific Computing at CSE 2009},
(Error may have been on previous line)
I'm skipping whatever remains of this entry
```

File: scipy-1.0/references.bib [hunk 1]
```diff
@@ -172,13 +172,13 @@ @article{peterson2009f2py
...
 
 @misc{siamcse09,
-  author="F. P{\'e}rez and H. P. Langtangen and R. LeVeque",
-  title="{CSE 2009: Python for Scientific Computing at CSE 2009}",
-  howpublished="SIAM News",
-  volume="42",
-  number="5",
-  year="2009",
-  url="https://archive.siam.org/news/news.php?id=1595"
+  author={P{\'e}rez, Fernando and Langtangen, Hans P and LeVeque, Randy}
```
Comment (ilayn) [inline scipy-1.0/references.bib]: missing a comma at the end of this line

**Comment by ilayn** [comment]
It's not a blocker but many of the URL's are actually the duplicates of the DOI identifiers so can be skipped if needed. But I'll leave it to you to decide.

**Comment by souravsingh** [comment]
Do we need to fix the author references for AstroPy? The reference was quite large so I left it as it is.

**Comment by mdhaber** [comment]
Reference 137 is only used in the Figure 3 caption. As discussed in #150, references only used in figure captions should be numbered sequentially _after_ all references first used in the body.

Reference 15 says just "van Rossum, G."

"Python" is not capitalized in several places.
"Fortran" capitalization is inconsistent. The [proper capitalization has changed](https://en.wikipedia.org/wiki/Fortran), so you'll need to check the actual references to see what they used, but "fortran" has never been wrong.
Please check capitalization throughout. The journal guidelines might say that article names should not have every word capitalized - please check - but still, proper nouns and acronyms need to be capitalized.

**Comment by mdhaber** [comment]
No, The Astropy Collaboration et al. is fine.

**Comment by mdhaber** [comment]
No response, so I started from scratch in #191

**Comment by souravsingh** [comment]
I had my notifications turned off. What needs to be done to finish the PR?

**Comment by rgommers** [comment]
finished in gh-191, closing. thanks @souravsingh

**Pull request closed without merging**